modules/caddyhttp/reverseproxy/streaming.go

```diff
@@ -65,17 +64,17 @@ func (h Handler) handleUpgradeResponse(rw http.ResponseWriter, req *http.Request
 
 	conn, brw, err := hj.Hijack()
 	if err != nil {
-		// p.getErrorHandler()(rw, req, fmt.Errorf("Hijack failed on protocol switch: %v", err))
+		h.logger.Sugar().Errorf("Hijack failed on protocol switch: %v", err)
 		return
 	}
 	defer conn.Close()
 	res.Body = nil // so res.Write only writes the headers; we have res.Body in backConn above
 	if err := res.Write(brw); err != nil {
-		// p.getErrorHandler()(rw, req, fmt.Errorf("response write: %v", err))
+		h.logger.Sugar().Errorf("response write: %v", err)
```

Comment:
Can be DEBUG level structured log, since writes can fail for client reasons, which I bet is more common.